projects/ad9213_dual_ebz/common/ad9213_dual_qsys.tcl

```diff
@@ -29,71 +33,49 @@ set_instance_parameter_value ad9213_rx_1 {SYSCLK_FREQUENCY} {100.0}
 set_instance_parameter_value ad9213_rx_1 {REFCLK_FREQUENCY} {320.3125}
 set_instance_parameter_value ad9213_rx_1 {INPUT_PIPELINE_STAGES} {2}
 set_instance_parameter_value ad9213_rx_1 {NUM_OF_LANES} {16}
+set_instance_parameter_value ad9213_rx_1 {EXT_DEVICE_CLK_EN} {1}
+
+# Link merger
+
+add_instance ad9213_dual_link adi_jesd204_link_merge
+set_instance_parameter_value ad9213_dual_link {NUM_OF_LANES_PER_LINK} {16}
 
 # ad9213_tpl_0 JESD204B transport layer
 
-add_instance axi_ad9213_0 ad_ip_jesd204_tpl_adc
-set_instance_parameter_value axi_ad9213_0 {ID} {0}
-set_instance_parameter_value axi_ad9213_0 {NUM_CHANNELS} {2}
-set_instance_parameter_value axi_ad9213_0 {NUM_LANES} {16}
-set_instance_parameter_value axi_ad9213_0 {BITS_PER_SAMPLE} {16}
-set_instance_parameter_value axi_ad9213_0 {CONVERTER_RESOLUTION} {16}
-set_instance_parameter_value axi_ad9213_0 {TWOS_COMPLEMENT} {1}
-
-add_instance axi_ad9213_1 ad_ip_jesd204_tpl_adc
-set_instance_parameter_value axi_ad9213_1 {ID} {1}
-set_instance_parameter_value axi_ad9213_1 {NUM_CHANNELS} {2}
-set_instance_parameter_value axi_ad9213_1 {NUM_LANES} {16}
-set_instance_parameter_value axi_ad9213_1 {BITS_PER_SAMPLE} {16}
-set_instance_parameter_value axi_ad9213_1 {CONVERTER_RESOLUTION} {16}
-set_instance_parameter_value axi_ad9213_1 {TWOS_COMPLEMENT} {1}
+add_instance axi_ad9213_dual ad_ip_jesd204_tpl_adc
+set_instance_parameter_value axi_ad9213_dual {ID} {0}
+set_instance_parameter_value axi_ad9213_dual {NUM_CHANNELS} {2}
+set_instance_parameter_value axi_ad9213_dual {NUM_LANES} {32}
+set_instance_parameter_value axi_ad9213_dual {BITS_PER_SAMPLE} {16}
+set_instance_parameter_value axi_ad9213_dual {CONVERTER_RESOLUTION} {16}
+set_instance_parameter_value axi_ad9213_dual {TWOS_COMPLEMENT} {1}
 
 # pack(s)
 
-add_instance util_ad9213_cpack_0 util_cpack2
-set_instance_parameter_value util_ad9213_cpack_0 {NUM_OF_CHANNELS} {2}
-set_instance_parameter_value util_ad9213_cpack_0 {SAMPLES_PER_CHANNEL} {16}
-set_instance_parameter_value util_ad9213_cpack_0 {SAMPLE_DATA_WIDTH} {16}
-
-add_instance util_ad9213_cpack_1 util_cpack2
-set_instance_parameter_value util_ad9213_cpack_1 {NUM_OF_CHANNELS} {2}
-set_instance_parameter_value util_ad9213_cpack_1 {SAMPLES_PER_CHANNEL} {16}
-set_instance_parameter_value util_ad9213_cpack_1 {SAMPLE_DATA_WIDTH} {16}
+add_instance util_ad9213_cpack util_cpack2
+set_instance_parameter_value util_ad9213_cpack {NUM_OF_CHANNELS} {2}
```

Comment:
Also here the channels should double, except if the mode changed from M=2 to M=1 per ADC.